src/lib.rs

```diff
@@ -1223,28 +1230,28 @@ mod tests {
         let _ = expression_is_literal(*rhs);
     }
 
+    // Is the operation that evalutes first in the LHS or RHS of the second operation?
+    enum FirstsLocation {
```

Comment:
I think this is a simple concept, but for some reason my feeble mind rebelling. Can you sketch an example here of both?

Reply:
Ah - this is test code. Less critical...but I still wanna know :)

Reply:
scroll down to see usages